cdc/filter.go

```diff
@@ -0,0 +1,37 @@
+// Copyright 2019 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cdc
+
+import (
+	"regexp"
+
+	"github.com/pingcap/ticdc/cdc/model"
+)
+
+func filterBySchemaAndTable(t *model.Txn) {
```

Comment:
how about return a bool value to mark whether the table should bu filter

Reply:
This is a function with side effect, it would remove mutations that belong to a database or a table that's in the black list.